manifests/m/momento/cli/0.26.3/momento.cli.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: momento.cli

```

```suggestion
PackageIdentifier: Momento.MomentoCLI
```
The convention for naming PackageIdentifier in WinGet is generally `Publisher.PackageName` (pascal case)